corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js

```diff
@@ -13,6 +13,14 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
 
     var CaseConfig = function (params) {
         var self = this;
+        self.help_name = gettext('Case Property Description');
+        self.default_description = gettext("You do not have a description for this case property. If you would like to add one you can do so in the data dictionary.");
```

Comment:
I actually think that currently is not used, and I was just leaving description blank for properties that do not have one. Happy to add a way to edit, though im not sure how it interacts with the fact that we truncate descriptions that dont fit inline and only show them in a popover. may have to offline with @orangejenny